src/data/markdown/docs/20 jslib/01 jslib/01 aws/00 SecretsManagerClient.md

```diff
@@ -5,11 +5,11 @@ description: 'SecretsManagerClient allows interacting with AWS secrets stored in
 excerpt: 'SecretsManagerClient allows interacting with AWS secrets stored in Secrets Manager'
 ---
 
-<BlockingAwsBlockquote />
+`SecretsManagerClient` interacts with the AWS Secrets Manager.
 
-SecretsManagerClient allows interacting with secrets stored in AWS's Secrets Manager. Namely, it allows the user to list, download, create, modify and delete secrets. Note that the SecretsManagerClient operations are blocking, and we recommend reserving their usage to the [`setup`](/using-k6/test-lifecycle/) and [`teardown`](/using-k6/test-lifecycle/) functions as much as possible.
+With it, the user can list secrets owned by the authenticated user, download secrets, create new secrets, update secrets, and delete secrets.
```

Comment:
Similar feedback as the previous one on having all the methods listed here.

Reply:
I'll rephrase too 👍🏻